### ISSUE TITLE
Screen reader does not announce the state of ‘New Section and Form Library’ tab items.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#105]
User Experience:

Current state of the element is an important information for the user to know and move further with the process, screen reader users will get impacted if state information is not provided via screen reader.

Note: User credentials should NOT be included in the bug.

 

Repro Steps:

1. Open Azure portal with valid credentials and create a API Management.

2. Click Developer Portal button to navigate to the portal.

3. Wait portal to load in design mode

4. Navigate to the Left nav and select Pages button.

5. Navigate to the Pages window. Select "APIs: List" option.

6. Navigate to the any editable control details and select Add section.

7. Enable screen reader and navigate to tab items.

8. Verify whether screen reader announcing state or not.

 

Actual Result:

Screen reader does not announce the current state  as selected or not selected of ‘New Section and Form Library’ tab items, present under 'New Section' popup.

 

Expected Result:

Screen Reader should  announce the current state as selected or not selected of ‘New Section and Form Library’ tab items, present under 'New Section' popup.

 

Note:

1. Similar Issue exist with Chrome +JAWS and Chrome + NVDA

2. JAWS and NVDA announcing incorrect state.

3. Same issue exist with scenario: Left Nav->Add widget->Edit Button->Link to->Hyperlink

4. Similar issue exist throughout the application for all tab items present in application.

MAS Reference: 

MAS 1.3.1 - Info and Relationships (40)

 

Reference Links

Accessibility Insights! - Identify accessibility bugs before check-in and make      bug fixing faster and easier.
External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries
